app/models/organisation.rb

```diff
@@ -21,10 +21,11 @@ class Organisation < ApplicationRecord
     uniqueness: {case_sensitive: false},
     presence: true,
     format: {with: /\A[a-zA-Z]{2,}-[a-zA-Z]{3}-.+\z/, message: I18n.t("activerecord.errors.models.organisation.attributes.iati_reference.format")}
+  validates :beis_organisation_reference, uniqueness: {case_sensitive: false}
   validates :beis_organisation_reference,
     presence: true,
-    uniqueness: {case_sensitive: false},
```

Comment:
Have we lost uniqueness or am I missing something?

Reply:
No, we validate by uniqueness on line 24. This validation only validates the format now